urbanairship-react-native/ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -87,12 +87,29 @@ - (instancetype)init{
     [[UAirship channel] enableChannelCreation];
 }
 
-RCT_EXPORT_METHOD(setDataCollectionEnabled:(BOOL)enabled) {
-    [[UAirship shared] setDataCollectionEnabled:enabled];
+RCT_EXPORT_METHOD(setEnabledFeatures:(NSArray *) features) {
+    [UAirship shared].privacyManager.enabledFeatures = [self parseStringFeatures:features];
 }
 
-RCT_EXPORT_METHOD(setPushTokenRegistrationEnabled:(BOOL)enabled) {
-    [[UAirship push] setPushTokenRegistrationEnabled:enabled];
+RCT_REMAP_METHOD(getEnabledFeatures,
+                 getEnabledFeatures_resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject) {
+    resolve(@([UAirship shared].privacyManager.enabledFeatures));
```

Comment:
I think you need to convert them back to strings right?

Reply:
Right 👍